integration/app/test-demo/src/constants/routes.ts

```diff
@@ -0,0 +1,11 @@
+const awsPath = '/Prod';
+export const rootPath: string = window.location.href.includes(awsPath)
+  ? `${awsPath}/`
+  : '/';
+
+const routes = {
+  HOME: `${rootPath}`,
+  SIMPLE_APP: `${rootPath}simple-app`,
```

Comment:
should we name this something more specific? If we have multiple tests, we want them to have specific scopes

Reply:
This one is just a simple-app, it has video/content share/meeting controls/roster/video tile grid. I tried to find a better name but not sure what was the best. Do you have any recommendation?

Reply:
This is a simple version of our meeting demo. Most of the integration test can running against this demo. For those tests that need specific setup, we can create specific demo for them and named it accordingly.

Reply:
Removed the "Simple App" and added "Roster Test App"